directus-django/integrating directus in django.md

```diff
@@ -0,0 +1,224 @@
+---
+title: 'Using Directus with Django'
+description: 'Django is one of the most popular python frameworks known for its battery included philosophy, it comes loaded with a lot of features that cuts down development time for  devs. In this tutorial, you will learn how to integrate django with Directus using a blog API. At the end of this tutorial, you will have an application that uses the django templating engine to display data from the API.
+'
+author:
+  name: 'omu inetimi'
+  avatar_file_name: 'omu-inetimi.JPG'
+---
+## Introduction
+Django is one of the most popular python frameworks known for its battery included philosophy, it comes loaded with a lot of features that cuts down development time for  devs. In this tutorial, you will learn how to integrate django with Directus using a blog API. At the end of this tutorial, you will have an application that uses the django templating engine to display data from the API.
+
+
+## Before You Start
+You will need:
+- Python installed and a code editor on your computer.
+- Directus project - Use the [quickstart guide](https://docs.directus.io/getting-started/quickstart) to create a project with the fields  title, content and author.
+- Basic knowledge of python and django
+
+## Project setup
+To start, open your terminal  and create an env, this will serve as a virtual environment for all necessary dependencies for this particular project. It's recommended you use an env for every project to ensure your project is isolated from others.
+
+```bash
+python -m venv myenv
+```
+Then go ahead and activate the virtual environment:
+- On Windows: myenv\Scripts\activate
+- On macOS/Linux: source myenv/bin/activate
+
+
+#### Installing Django
+With the virtual environment activated, install Django within this env:
+
+```bash
+pip install django
+```
+
+This installs django within the virtual environment.
+
+#### Initializing Django Project
+Create a new Django project and move into the directory of the new project
+
+```bash
+django-admin startproject myblog
+cd myblog
+```
+
+This command starts a new django project and loads up the boilerplate files within django
+
+To fetch data from the Directus API, we will use the *requests* library. It is not included in Python’s standard library, so it needs to be installed separately:
+
+```bash
+pip install requests
+```
+We need it to be able to make http requests to external API’s.
+
+
+#### Creating the Blog App:
+Create a new app within your Django project to handle the blog functionality:
+```bash
+python manage.py startapp blog
+```
+This creates a new directory within our project containing all the necessary files for a django app
```

Comment:
This isn't repeating what was before, but let's consolidate it with the previous sentence before the command.